### ISSUE TITLE
Use ft.layers instead of tf.contrib.slim

### ISSUE THREAD

**Issue by blazekadam** [Cognexa/cxflow-tensorflow#113]

**Comment by FloopCZ** [comment]
:+1: but I would prefer tf.layers.

**Comment by petrbel** [comment]
@gdynusa is this still relevant? Please explore the project and try to find out what `slim` lines could be replaced with regular `tf.layers`.

**Comment by blazekadam** [comment]
@petrbel not really, current implementation heavily depends on `slim.argscope` which does not have a counterpart in `layers` although there are some attempts in `contrib`. Escaping `contrib` is the reason for switching to `layers` in the first place though...

**Comment by petrbel** [comment]
So this issue can be closed? Or are we waiting for layers implementing the required features?

**Comment by blazekadam** [comment]
I would keep it open. It is just more complex than replacing `tf.contrib.slim` with `tf.layers`.
